packages/repository/src/__tests__/fixtures/mixins/categoryPropertyMixin.ts

```diff
@@ -0,0 +1,26 @@
+// Copyright IBM Corp. 2020. All Rights Reserved.
+// Node module: @loopback/repository
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {Constructor} from '@loopback/context';
+import {property} from '../../../index';
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export function AddCategoryPropertyMixin<T extends Constructor<any>>(
```

Comment:
Can we use `Constructor<Entity>` to avoid no-explicit-any problem? I think such type is more correct than `Constructor<any>`.